oeps/processes/oep-0056-architectural-advisory-process.rst

```diff
@@ -0,0 +1,197 @@
+
+We will necessarily iterate on the organization of the checklist as we learn more about how we use it.  This OEP also does not prescribe where the checklist needs to be written down.
+
+Other Options Considered
+========================
+
+Synchronous Architecture Advisory Forum meeting
+-----------------------------------------------
+
+The approach to the Advisory Process advocated for in `Scaling the Practice of Architecture, Conversationally <conversationally_>`_ centers around the "Architecture Advisory Forum", a synchronous meeting where the group reviews ADRs, Architectural Principles, Tech Radar blips, etc.
+
+We see three significant problems with the idea of having such a forum:
+
+- Our community is global.  Any time we could propose for this meeting would necessarily leave out a significant portion of our stakeholders and experts.
+
+- Our community is multi-faceted.  ADRs may have completely divergent sets of stakeholders.  How do we manage the guest list of such a meeting to ensure that the right voices are speaking on any given ADR, and that the right individuals are in the room?  It's a logistical nightmare given the size and scope of our community.
+
+- Our community is comprised of many independent organizations.  There are times when our component organizations may want to use the Advisory Process on "internal" decisions as well.  Do we create a separate forum meeting for those decisions?  How do we then ensure that we err on the side of having an open, public forum?  Again, logistically, this is quite difficult.
+
+.. _principles:
+
+Appendix A: Advisory Process Principles
+=======================================
+
+The creation of the Advisory Process is a fundamental shift in expectations of how we conduct our architectural process, and that shift requires an investment of time in order to be successful. The principles below describe the expectations, spirit and benefits of this process.
+
+Asynchronous First
+------------------
+
+Our community is global, and so we will prioritize asynchronous communication in the Architecture Advisory Process. This means it's important that individuals be able to participate in the process even if they cannot attend a synchronous meeting.  We believe there's value in getting together synchronously, but the process should prioritize getting the right input over having synchronous meetings.  That said, if the relevant stakeholders and experts are able to schedule an effective in person meeting, there's additional value in getting together synchronously when possible.  Note, however, that the decision makers *still have the responsibility of recording in their ADR the advice they received.*
+
```

Comment:
would there be some sort of term for this asyncronous participation? In order to make it faster and not keeping everything blocked?

Reply:
Hm, I think we should consider asynchronous participation the primary form, and synchronous meetings the exception.  I could probably make that clearer here.  

I was trying to indicate that if it makes sense - given the participants - to try something synchronous, the process shouldn't preclude that, since it can be very effective.

Reply:
I was thinking abot like writing a target date to contribute async. Similar to what we do for voting in the CC program but shorter.

Reply:
+1 to encouraging deadlines and providing guidance around them.